scribejava-apis/src/main/java/com/github/scribejava/apis/FlickrApi.java

```diff
@@ -21,6 +30,7 @@ public static FlickrApi instance() {
         return InstanceHolder.INSTANCE;
```

Comment:
perms is not required, things will default to read only if you don't ask for one